circuit-lib/circuitlib-rs/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "circuitlib-rs",
```

Comment:
if we move the circuits to circuit-lib.circom we probably don't need a package json in circuit-lib.rs